packages/gestalt/src/useResponsiveMinWidth.js

```diff
@@ -0,0 +1,69 @@
+// @flow strict
+import { useMemo, useCallback, useState, useEffect } from 'react';
+import { addListener, removeListener } from './utils/matchMedia.js';
+
+type MinWidthType = 'lg' | 'md' | 'sm' | 'xs';
+
+const breakpoints = {
+  sm: '(min-width: 576px)',
+  md: '(min-width: 768px)',
+  lg: '(min-width: 1312px)',
+};
+
+export default function useResponsiveMinWidth(): ?MinWidthType {
```

Comment:
would "getResponsiveMinWidth" make more sense than "useResponsive..."?

Reply:
good point... it's intended to simulate the hook naming pattern. 

useColorScheme
useReducedMotion
useFocusVisible
useDebouncedCallback
etc

I'm not exposing it yet, so we can discuss this on the next meeting.
I think for now works well. Maybe we can discuss further the naming all the different hooks, utils we are building.